projects/storefrontlib/src/shared/components/spinner/spinner-button.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+@Component({
+  selector: 'cx-spinner-button',
+  templateUrl: './spinner-button.component.html',
+})
+export class SpinnerButtonComponent {
```

Comment:
Still not sure about the name. If it could be just ButtonComponent or something else. I'll need to discuss and think about that more.